crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -279,12 +323,19 @@ impl<'a> SimpleTokenizer<'a> {
             '\\' => TokenKind::Continuation,
 
             c => {
-                let kind = TokenKind::from_non_trivia_char(c);
+                let kind = if is_identifier_start(c) {
+                    self.cursor.eat_while(is_identifier_continuation);
+                    let token_len = self.cursor.token_len();
+
+                    let range = TextRange::at(self.offset, token_len);
+                    self.parse_identifier(range)
```

Comment:
Nit: The method name indicates to me that this parses an identifier and not a keyword. I haven't been able to come up with a name that I like, but was thinking of `match_keyword` or `to_keyword_or_other`

Reply:
sure, i don't mind. i also considered making it a method on `TokenKind` (similar to `from_non_trivia_char`)

Reply:
That would work too!